docs/endpoints.json

```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.0",
   "servers": [
     {
-      "url": "http://localhost:5005"
+      "url": "http://0.0.0.0:3000/api"
```

Comment:
what does this do? red flag is only stating /api when we have multiple other endpoints.

Reply:
This is mostly a carry-over from manually writing that file. It doesn't reflect all the `tags` in the actual implementation. It specifies the base url and port. It should be replaced with an automatically generated file based on the code.